app/views/shared/_service_comparison.html.erb

```diff
@@ -0,0 +1,360 @@
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+            <div class="service diy">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+            <div class="service express">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+          </div>
+        </div>
+        <div class="service-set">
+          <div class="service-set__header">
+            <div class="service-set__title service-set__title--normal">
+              <%= t("views.shared.service_comparison.stimulus_payments") %>
+            </div>
+          </div>
+          <div class="service-set__services service-set--mobile-excluded">
+            <div class="service full-service">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+            <div class="service diy">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+            <div class="service express">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+          </div>
+        </div>
+        <div class="service-set">
+          <div class="service-set__header">
+            <div class="service-set__title service-set__title--normal">
+              <%= t("views.shared.service_comparison.earned_income_tax_credit") %>
+            </div>
+          </div>
+          <div class="service-set__services">
+            <div class="service full-service">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+            <div class="service diy">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+            <div class="service express">
+            </div>
+          </div>
+        </div>
+        <div class="service-set service-set--hr">
+          <div class="service-set__header">
+            <div class="service-set__title service-set__title--normal">
+              <%= t("views.shared.service_comparison.state_credits") %>
+            </div>
+          </div>
+          <div class="service-set__services">
+            <div class="service full-service">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+            <div class="service diy">
+              <%= image_tag "icons/check-outline.svg", alt: "yes" %>
+            </div>
+            <div class="service express">
+            </div>
+          </div>
+        </div>
+        <div class="service-set">
+          <div class="service-set__header">
+            <div class="service-set__title service-set__title">
+              <%= t("views.shared.service_comparison.additional_benefits") %>
+            </div>
+          </div>
+          <div class="service-set__services service-set--mobile-excluded">
+            <div class="service full-service">
+            </div>
+            <div class="service diy">
+
```

Comment:
This blank line in the div freaks me out and makes me wonder if it's meaningful (`white-space: pre-wrap` or something). I'm going to assume it's not meaningful. Not a blocker.